label.go

```diff
@@ -19,21 +28,23 @@ type Label struct {
 	Comment string `nebula:"Comment"`
 }
 
+type LabelSchema struct {
+	Name        string
+	Fields      []LabelFieldSchema
+	TTLDuration uint
+	TTLCol      string
+}
+
 type LabelFieldSchema struct {
 	Field    string
 	Type     string
 	Nullable bool
```

Comment:
add defaut value

Reply:
can we add this in another PR?
may need some tests and changes for this (`default` supports)

Reply:
Of course you can.

Reply:
I created a issue for this 
https://github.com/vesoft-inc/nebula-go/issues/330